examples/seismic/accuracy.ipynb

```diff
@@ -0,0 +1,433 @@
\nqoA7e1YGVjQqA14yWgo4cmQxEpSKIAN0ADjAkSPFPPDAEvbuhXXr/pqqNWuouesulu/aRa619mHH\ncZRfffAAABdhSURBVO2x/oJYE1JEo6z48Y+ZU14uXk51NRWrVlERDhMKlSS8c9UOVynNWMyd0FVD\nrrffHq3InX9vv518PDLtrpgNnATuVvqur+uWFujVEtZ5AcjLs/Pu586V31J53RX+U8lNdR5/nDM9\nPRxAyPsgo+e2VKhClHcjMOuGG2DDBg7nLmfdOrnQHzlyBDuWAjAb0etBliwpIBwmMf2f8r7LyuzU\n11Tq23k+68/183o840d9Trb7dW6X7rtdKHjHKHGF8SpyhUyv5E54/fiZqAt9Pef3UXB65dGoFAe9\n+aZdut/aKuP/pZdiyGDtQ4b+MLYyX8GSJX6++lVobIRLW/8Tdu/m+KOP8jrimfcy2lsFCXUFkcyG\nEOKjz1q2DD70IbvuOhzmlL+C/n5boTvbocLoyk5nOqKTMHTiTuefZ0Pmbmpbf65bHak6COrbOP83\ntzbAiiRLokflR1N9F556inOHDtGB/A6dCGk7U0VBlFoI8bwXA/M+/WlYtYqX6z5DUxM8+KBS3gew\nlXc+cn9VDBRz+eV5SeRdXS3fU5F3Ku8bMjt/06lkhVRjK9MxeSErcAWjxD0wFYp8LOTuXMfrO+nP\n3QaDk7gUqc2eLeQRDMpjKAQjI3lEIvPo6ytCUsVUsdAwujJvaoJ77vkM19x8M/OCQebt2sWcjg4i\nwMuM7skyoP0FER82dOgQyw8dYlZlpUQ+L76Ykuuuo6S8nPC1y+nrEwKPRJItF9Xnw6nWnb63W9DT\n61h4Za3o5OPlfXspb69ME+e+nCobkjtFBgKitufOlUBlftdBaO+RKPVrr8GuXZzr6uIgyeTtpboD\nWHNbAitqasTiuv9+nmudwwPrxGrr7lbKW+82WABcTHGxP4m8a2rsyZEDATt10K3zIIxOrdWfp7Mk\nx0PW7wQFrvCOU+IK2SryTPy3VPsfy6Pb/r2eq9c6FHnohSFDQ6LG+/ul+vLkSXm9f7/KZDmGDGYV\nCFWBrGoKC4u5/XYpof6/Gg5L2sK6dfR3dxNBfHPVKdENQeuvEnvygKK8PGhstOf9uvJKkXc1NQz7\nixKk3ttrf3+3gJ+CW7WngpfyTqXIMw1ken2uWzaOytoIhWzSzo8OyMXtxAnpHKjm0WtqYiAWS0z2\n0UVq0i5CVPcVCHEHFy2CdeugsZGfNFfR3CwFmoODfcgl4DRyAc9H+nwXU1i4gIULpUtxebn43MuW\nJZO3WytiHc7zMhMbZbzjJVPhNFYyn2oCN0o8DcaiyNMtc+5DX19/zPaEzIbIdeheuVO5FhTIY1mZ\nDNKREViwAFpaFjD4/7d3trFxHOcd/43IO11I6nyUKImySYtSLEGx45guVFuB1MIxHMANhAIpYqBp\ngCao036pgRhoUNgoUKAfixiNW7go2qSBPqRtCreREwgIAslxPtiI5JeKDu1YsiRYjmSLFimRFk/S\nWUdq+2H2uZ0bzr6RlI6nmz9wmN3Zt9m5nf/+59lnnpntR5O36Olp4Dizs2Wee+5uDh6EiT/fyu7d\nW3lw3wiVX/6SyssvU3npJT5Eq/OLRCFwBUI+F9EKcQKo1OuMHDxIBShVKnqo/9AQ3HcfxU2b2Lpj\nBwwOcOWuDczO6skrxOxiBncSUpd7dZGv6epoIo/fuG0CEdg+7jZpi11bJltYswZ6quf12+nAMTh3\nDt58U5P3r35FbWamibR/i+7xxL0ge9Dxbz5H6G3yhS9os9VDD3Gk92FeeUEPkT9+HILgN0T/rcyy\nUwY2s2ZNiZ07NVnfc49+PiqVaNCOOWlDFq8TW8may3bbyNMu0tqTfR2XGSeLkLPLuhLRsUpckPWP\nhORAPItRBUnmlTxmFXtZ1m3YnhGXL0e26NOnNTEeO6aV+dGjmjCnp8+jG/tHQJ1Iten40Pfeq93M\nRkfhiSegOPYqPPMMHD7Mu2fONPySL+L2bhH0oIn9djSdmCmDg3D//Vq6iiOyxDENHZKvdJcbA41M\nU4ztaw3Rx0WpkzSY5hjbfm2HFjBJrmfuUuTAL/GExZF/akpX8sQEH4Y1bKZVFr4ETZTQpC1++9uH\nh3UX6dvf5troAzz3nP72MTYG4+MSrli+e3Sh49FvBMr0929g40ZdxWIyue023TkS23zaiEvIp77N\n5TRTY1y7yWNOsY9L61G3UoELvBLPiLyKPMu2uHMtRYG7lIG53e4Suh5Us+GJPVb8yufntQ0WtFLs\n7paPoBuYnFyHbvhVIjPLOWCK8fEhxsc3cuhQidOnYXT0Af5s3z44fJjtzz7L9tdeY3Biggkijwnb\nhg6asK6gaaYH/YroQxNU38QEgz/7GX1An3QdPvMZTeqbN0OlQs/wMD1hP/92sVNUKlzvLi4IDpXk\nEWIiyfXRnOFo1dy1KPD7WcPuc+aMzn//fU3a77wDFy5QnZykiu6JmOl5kombsE4q6EmIB4Htg4Pa\n7vHkk7BrFz/4UQ9jP9SDM8+dq6FfvmeJXsBdaLNJH3An69d3Nezd99yjn4GREX1f8tGyqyuf14m9\n7lLAss9SFHkW5b2Ydmyfa6Wj45W4IOsfm0beaYo87tg0Uk/qFiblm+s2bGX+ySeac65e1fbxmRkt\nHiXI1sSEeDFcQpP4VZptqjoIUqm0hZEReOwxrdD/6JFL+gva978Pr7/O2RMnmCIi9Q+dpVuIIlp9\nlhLSItBXKum+/+CglpDCRiKTTQaGyLBrDuUUqS7y3Z4SfmYmcv25fFlXzuws1VqtEbHmYkJqzo6T\nBO3Yp0l7ABjatk07Z3/zm7BzJz8+VGZsDJ5/XvemajUZnTtN8zeNT6Hjm5TZsqXE4GAUnOquuyKT\niYRzsGeajzObpJF3Uv5iSdtG3raYpOpXgg
```

Comment:
can you use the ``e-0.." notation for all yvalues ? And is it possible to decrease the number of precision digits? the yaxis is really verbose ATM

Reply:
Yes, Still digging in opescibench to get nicer plots.